scalableminds/webknossos
# Add Pricing Plans to Organization Page

This PR overhauls the organization page to include more infos on the current number of users, storage quota, and the selected webKnossos plan.

Changes:
- Redesigned Organization Page
- Enforced Limits for User Invites
- Warning for exceeding your plan limits soon on the dashboard
- several modals to upgrade plan, users, storage and time that can be re-used all over the place

PR does NOT yet enforce any webKnossos annotation features / tools depending on your plan. This will come in a follow-up PR.

ToDo:
- [x] send emails on upgrade

### URL of deployed dev instance (used for testing):
- https://pricing.webknossos.xyz

### Steps to test:
- Extensively monkey test the orga page
- Make sure to try all sorts of the combinations of Plans, custom storage and user quota etc in the DB

### Issues:
- contributes to #6509 

------
(Please delete unneeded items, merge only when none are left open)
